aerleon/lib/policy_builder.py

```diff
@@ -923,4 +931,4 @@ def _NormalizeValues(cls, context: RecognizerContext, repr):
             # by the user.
             repr = (repr['name'], repr.get('policy', ''))
 
-        return repr
+        return repr
```

Comment:
Nitpick: final newline is missing. Also the indentation on the data changes to _Builtin and _BuildTerm do not match the surrounding lines.